WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -84,6 +84,21 @@
   </div>
 </div>
 
+<!-- Hidden data range feature enabled by setting one of those parameters in the URL. -->
+<% if %i(date_range from_date to_date).any? { |key| params.key? key } %>
```

Comment:
That is true. Why is this a problem, though? Do our datepickers submit localized dates to the backend?